static/css/around-map.css

```diff
@@ -34,27 +47,58 @@
   align-self: flex-end;
 }
 
-.first-card {
-  order: 0!important;
-  border: 0.2rem solid #3BA77C!important;
+/* ================================================================= */
+/* OSM & Preact App build CSS and erase below without !important*/
```

Comment:
When you explained to me you were more saying that the code is written onto the HTML tag and can't be overrwritten unless using `!important`.